src/celeritas/em/data/GammaNuclearData.hh

```diff
@@ -61,16 +55,19 @@ struct GammaNuclearData
     // Scalar data
     GammaNuclearScalars scalars;
 
-    // Microscopic (element) cross section data (G4PARTICLEXS/gamma/inelZ)
-    ElementItems<NonuniformGridRecord> micro_xs;
+    // Microscopic cross sections using G4PARTICLEXS/gamma nuclear (IAEA) data
+    ElementItems<NonuniformGridRecord> micro_xs_iaea;
+    Items<real_type> reals_iaea;
```

Comment:
You should be able to use the same `reals` backend array for both grid types.

Reply:
Used a common backend array for both cross section grids.